cms/server/admin/handlers/user.py

```
@@ -94,6 +96,49 @@ def post(self, user_id):
+                    try:
+                        processed_data, _ = process_picture(
+                            picture_file["body"],
+                            picture_file["content_type"]
+                        )
+                        attrs["picture"] = self.service.file_cacher.put_file_content(
+                            processed_data,
+                            "Profile picture for %s" % attrs.get("username", "user")
+                        )
+                        new_picture_uploaded = True
+                    except PictureValidationError as e:
+                        raise ValueError(e.message)
```

Ruff (0.14.13)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)